pdfminer/layout.py

```diff
@@ -675,6 +718,10 @@ def analyze(self, laparams):
         # textobjs is a list of LTChar objects, i.e.
         # it has all the individual characters in the page.
         (textobjs, otherobjs) = fsplit(lambda obj: isinstance(obj, LTChar), self)
+        if laparams.split_tables:
+            (splitobjs, otherobjs) = fsplit(lambda obj: isinstance(obj, LTRect), self)
```

Comment:
Why do you only select `LTRect`? Are all tables composed of rectangles? Are there other literals that could indicate that two textboxes belong together?